psl-core/src/main/java/org/linqs/psl/config/Options.java

```diff
@@ -327,6 +342,18 @@
         "A comma-separated list of possible weights. These weights should be in some sorted order."
     );
 
+    public static final Option WLA_GS_POSSIBLE_ANGLES = new Option(
+            "gridsearch.angles",
+            "0.0001:0.3926:0.7854:1.1781:1.5708",
+            "A comma-separated list of possible weights. These weights should be in some sorted order."
```

Comment:
The variable name and key refer to angles, but here you refer to weights.
You need to to reconcile these terms.